glutil/partitioner.py

```diff
@@ -231,16 +231,17 @@ def _partition_input(self, partition):
     def bad_partitions(self):
         """Return a list of bad partitions
 
-        Bad partitions are defined as any partition that exists in AWS, but do
-        do not exist on disk, or exist on disk in a different location.
+        Bad partitions are defined as any partition that exists in the
+        Glue Data Catalog, but do not exist on disk, or exist on disk
+        in a different location.
 
         For example, a partition with the values [2019, 01, 01, 01] exists in
-        AWS with the location 2019/02/02/02 is considered bad. Similarly, a
-        partition that exists in s3://another-bucket/, while the table's
+        the Glue Data Catalog with the location 2019/02/02/02 is considered bad.
+        Similarly, a partition that exists in s3://another-bucket/, while the table's
         location is set to s3://this-bucket/ is also considered bad.
 
         Returns:
-            list of Partition
```

Comment:
This should stay `list of Partition`, because it returns a list of `Partition` objects.